packages/InteractiveProfilingTool-Core.package/IPTMessageTallyWrapper.class/instance/getRootOverflows.st

```diff
@@ -0,0 +1,6 @@
+information extraction
+getRootOverflows
+	
+	self messageTally gcStats ifNil: [^ nil].
```

Comment:
code duplication with `getMemoryStats`. I would suggest adding the function
```
hasNoGcStats
    ^ self messageTally gcStats isNil
```
And change this line (and the line in `getMemoryStats`) to
```smalltalk
self hasNoGcStats
   ifTrue: [^ nil]
```